blogposts/our-journey-to-all-remote.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Our journey to all-remote"
+author: Nick Snyder
+publishDate: 2020-03-30-T10:00-07:00
+tags: [
+  blog, remote
+]
+slug: our-journey-to-all-remote
+heroImage: /blog/sf-office-mural.jpg
```

Comment:
@ryan-blunden is this the right place to put a hero image or should it also be in a Google Cloud Bucket?

Reply:
Yes, that's the correct path. And the location of the image should be: `about/website/static/blog`.

Reply:
@nicksnyder We may look at using a Google Cloud Bucket exclusively at some point, but this is fine for now.